### PR TITLE
Created simple sampler and inverted pendulum model

### PR DESCRIPTION
**Pull request by DomBaril**
To review

### CODE REVIEW

File: src/barebones_mpc/model/inv_pendulum_model.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+        y: cart position
+        v: cart velocity
+        theta: pendulum angle
+        q: pendulum angle rate of change
+        u: force applied to the cart
+
+        """
+        self.input_dimension = 1
+        self.state_dimension = 4
+        self.time_step = time_step
```
Comment (RedLeader962) [inline src/barebones_mpc/model/inv_pendulum_model.py:16]: Are we going to expose `time_step` as a parameter in every model. If so we are going to make a small change (in a later minor refactoring task) to the __init__ of the baseclass and every child subclass

Comment (DomBaril) [reply src/barebones_mpc/model/inv_pendulum_model.py:16]: I feel like time_step is quite clear as variable name so yes I would go with that. I am open to other suggestions though.

**Comment by RedLeader962** [comment]
Good to go! It's taking shape 🔥

**Review by RedLeader962 — APPROVED** [review]
Thumbs up 👍🏻

**Pull request merged**